src/Stratis.Features.FederatedPeg/Wallet/FederationWalletManager.cs

```diff
@@ -156,7 +246,10 @@ public void Start()
             {
                 // Find the wallet and load it in memory.
                 if (this.fileStorage.Exists(WalletFileName))
+                {
                     this.Wallet = this.fileStorage.LoadByFileName(WalletFileName);
+                    this.RemoveUnconfirmedTransactionData();
```

Comment:
Does moving this to here rather than CCTS have anything to do with this issue? No issues just wondering

Reply:
No.